realm/realm-library/src/androidTest/java/io/realm/RealmAsyncQueryTests.java

```diff
@@ -194,7 +194,7 @@ public void onSuccess() {
             @Override
             public void onError(Throwable error) {
                 // Ensure we are giving developers quality messages in the logs.
-                assertEquals("Could not cancel transaction, not currently in a transaction.", testLogger.message);
+                assertEquals("Could not cancel transaction, not currently in a transaction.", testLogger.previousMessage);
```

Comment:
There are two warning messages. The previous one is for `Could not cancel transaction, not currently in a transaction.` and the current one is for `closed with pending async transactions or callbacks.`.